controller/controller.go

```diff
@@ -245,15 +251,17 @@ func ConfigureNetwork(model *model.SystemInstall) error {
 	prg, err := configureNetwork(model)
 	if err != nil {
 		prg.Success()
+		NetworkPassing = false
 		return err
 	}
+
+	NetworkPassing = true
+
 	return nil
 }
 
 func configureNetwork(model *model.SystemInstall) (progress.Progress, error) {
-	if model.HTTPSProxy != "" {
-		cmd.SetHTTPSProxy(model.HTTPSProxy)
-	}
+	cmd.SetHTTPSProxy(model.HTTPSProxy)
```

Comment:
what happens if a proxy is not set?

Reply:
If it is empty, it is set to empty and works.
I've tested this code both with a value, empty, new value and back to empty -- works.
In fact, there was a corner case where the old value was left because even though the model was updated to be empty, the cmd still had the old proxy value.